lib/ui/semantics.dart

```diff
@@ -225,6 +225,8 @@ class SemanticsFlag {
   static const int _kIsInMutuallyExclusiveGroupIndex = 1 << 8;
   static const int _kIsHeaderIndex = 1 << 9;
   static const int _kIsObscuredIndex = 1 << 10;
+  static const int _kIsEdgeIndex = 1 << 11;
```

Comment:
Why not isRoute and isRouteName? Makes it clearer that the two belong together and I have an immediate idea what they mean. I don't think we use the term "edge" anywhere else in the framework with this meaning?

Reply:
Fixed